llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -2743,10 +2750,11 @@ multiclass VPseudoVSSHT_VV_VX_VI_RM<Operand ImmType = simm5, string Constraint =
   }
 }
 
-multiclass VPseudoVALU_VV_VX_VI<Operand ImmType = simm5, string Constraint = ""> {
+multiclass VPseudoVALU_VV_VX_VI<Operand ImmType = simm5, string Constraint = "",
+                                bit Commutable = 0> {
   foreach m = MxList in {
     defvar mx = m.MX;
-    defm "" : VPseudoBinaryV_VV<m, Constraint>,
+    defm "" : VPseudoBinaryV_VV<m, Constraint, Commutable=Commutable>,
```

Comment:
Nit, I guess we could also wrap the VPsuedoBlahV_VV in isCommutable so we don't need to add the argument to the _VV classes

Reply:
Yeah, it may make sense. I added this argument just to match existed code.